.github/workflows/build.yml

```
@@ -95,6 +106,18 @@ jobs:
 
       - name: Integration test on dotnet 7.0
         run: dotnet test ./NewPlatform.Flexberry.ORM.IntegratedTests/bin/Debug/net7.0/NewPlatform.Flexberry.ORM.IntegratedTests.dll
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)